### PR TITLE
Changes for Right Sizing implementation of AWS SSO

### PR DESCRIPTION
**Pull request by ganeshjagdale9545**
This changes required for Right Sizing implementation of AWS SSO

### CODE REVIEW

File: cartography/intel/aws/iam.py [hunk 1]
```diff
@@ -718,6 +718,33 @@ def _load_policy_tx(
         aws_update_tag=aws_update_tag,
     )
 
+    ingest_sso_policy = """
```
Comment (mpurusottamc) [inline cartography/intel/aws/iam.py:721]: we are inserting the same policy again?

**Review by mpurusottamc — COMMENTED** [review]
code looks okay. but, let's chat about the changes. I have some questions around the move of permissions sets related code move.

**Review by mpurusottamc — APPROVED** [review]
lgtm

File: cartography/intel/aws/iam.py (hunk 1 above)
Comment (ganeshjagdale9545) [reply cartography/intel/aws/iam.py:721]: in sso scenario we mapping the principal with id field and in non sso scenario mapping with arn, that’s why we need to separate queries

**Pull request merged**